multimodal/src/autogluon/multimodal/learners/matching.py

```diff
@@ -221,6 +221,8 @@ def __init__(
         self._total_train_time = None
         self._best_score = None
 
+        self._log_filters = []
```

Comment:
Matcher shares the same `run_trainer` API with base learner. Using an empty log filter is overwriting the log filters in base learner.

Reply:
Fixed with correct filters. Matcher did not call `super.__init__`, so we'll need to initialize `self._log_filters`.